<?xml version="1.0"?>
<launch>
  <include file="$(find navigation_prototypes)/launch/frames.launch" />

  <!-- invisible map requires a keyboard press in order to switch between calibration and run modes -->
  <node pkg="keyboard" type="keyboard" name="keyboard" />

  <node pkg="navigation_prototypes" type="nav_waypoints.py" name="nav_waypoints" output="screen" />

</launch>